integration-test/helpers/pages.js

```diff
@@ -578,13 +578,19 @@ export function loginPageMultistep (page, server, opts) {
  * A wrapper around interactions for `integration-test/pages/email-autofill.html`
  *
  * @param {import("playwright").Page} page
- * @param {ServerWrapper} server
+ * @param {ServerWrapper} [server]
  */
 export function emailAutofillPage (page, server) {
     const {selectors} = constants.fields.email
     return {
-        async navigate () {
-            await page.goto(server.urlForPath(constants.pages['email-autofill']))
+        async navigate (domain) {
+            const emailAutofillPageName = constants.pages['email-autofill']
+            if (domain) {
+                const pagePath = `integration-test/pages/${emailAutofillPageName}`
```

Comment:
This will only work for the extension pages etc - but what about other tests that use `emailAutofillPage`? Will this test need updating then?

Reply:
@shakyShane Not sure I follow -- only in-context signup integration tests pass the `domain` value, so all other tests which use this page object will remain unchanged.

This code shouldn't need changing when we update other tests which use this page object and the mocked flow -- they just need to pass in a domain value. However, those changes can be part of a follow-up and won't be in this PR.

Reply:
I've approved, but just to round off the conversation:

The reason I questioned this, is that we only have 2 folders to serve files from, so having it check in both would've solved for now + all future changes too. But as you say, it can follow 👍🏻